HW04.Rmd

```diff
@@ -29,34 +33,73 @@ Knitr is part of the knitr package. Allows you to change column names and add ca
 
 
 ```{r month, warning=FALSE}
+deaths_per_month = gun_deaths %>%
+    count(month)
+kable(deaths_per_month, format = "markdown")
 
 ```
 
 
 ### Generating a bar chart with human-readable labels on the x-axis. That is, each month should be labeled "Jan", "Feb", "Mar" (full or abbreviated month names are fine), not `1`, `2`, `3`.
 
 ```{r month_chart}
+month_labels = c("Jan", "Feb", "Mar", "Apr", "May", "Jun",
+               "Jul", "Aug", "Sept", "Oct", "Nov", "Dec")
+deaths_per_month %>%
+    mutate(month_lab = month_labels[month]) %>%
+    ggplot(aes(x = reorder(month_lab, month), y = n)) +
+    geom_bar(stat = "identity")    # 'identity' means you've already done the summarization
+    #scale_x_continuous(breaks = 1:12, labels = month_labels)
+
+# The easier way do to exactly the same thing:
+# Set 'month' as x aesthetics, and add the last commented line
+# However, it is fun to achieve this by mutating the original tibble
 
 ```
 
 ## Generating a bar chart that identifies the number of gun deaths associated with each type of intent cause of death. The bars should be sorted from highest to lowest values.
 
 ```{r intent}
+deaths_by_intent = gun_deaths %>%
+    count(intent)
+deaths_by_intent %>%    # Not sure why cannot pipe here

```

Comment:
I don't think you need to pipe here at all. You should be able to start directly with a ggplot using the modified data set "deaths_by_intent"

Reply:
That's true. I don't need to, but it is still weird that piping would give unexpected results.